### DISCUSSION TITLE
Maxed out config

### DISCUSSION THREAD

**Discussion by Wulfheart** [tailwindlabs/tailwindcss#3543] [Help] [answered]
I'd like to test the speed of some build tools. Is there any maxed out config file?

**Comment by adamwathan** [comment, marked as answer]
This should be pretty ridiculous:

```js
const colors = require('../colors')

module.exports = {
  darkMode: 'media',
  theme: {
    colors: {
      transparent: 'transparent',
      current: 'currentColor',
      ...colors,
    }
  },
  variants: [
    'responsive',
    'dark',
    'first',
    'last',
    'odd',
    'even',
    'visited',
    'checked',
    'group-hover',
    'group-focus',
    'focus-within',
    'hover',
    'focus',
    'focus-visible',
    'active',
    'disabled'
  ]
}
```

Hope you have a lot of hard drive space 😅

**Comment by adamwathan** [comment]
I think you'll find the bigger you make the Tailwind config, the less the build tool influences the build time though. Tailwind itself is going to be doing the same work no matter what build tool you use, and with a config this big, it will be the source of 99% of the build time.

You might be better off just using the default config, guess it depends on what you're trying to measure. Webpack for example is very slow when including _any_ large CSS file (even a 3mb CSS file that is totally static with no PostCSS processing at all).
